models/Element/AdminStyle.php

```diff
@@ -49,7 +49,7 @@ class AdminStyle
     /**
      * @param AbstractObject|Asset|Document|ElementInterface $element
      */
-    public function __construct(ElementInterface $element)
+    public function __construct(ElementInterface $element, Translator $translator)
```

Comment:
BC break as this can be extended in Custom DataObject classes https://pimcore.com/docs/pimcore/current/Development_Documentation/Tools_and_Features/Custom_Icons.html#page_Example-Custom-Icon-for-the-Car-DataObject

Reply:
I'm wondering why it doesn't work in your case, as the alias is there https://github.com/pimcore/pimcore/blob/417b50202ceba7721311eb73f92c308f84236546/bundles/CoreBundle/Resources/config/aliases.yaml#L13

Reply:
I have seen the alias, but it does not seem to work in my case. I have commented/uncommted it, without any effect!?

Reply:
please try this on your console and check the output:
```cmd
bin/console debug:container Symfony\Contracts\Translation\TranslatorInterface
```
It should say it is a public  alias for service `Pimcore\Translation\Translator`

Reply:
<img width="690" alt="image" src="https://user-images.githubusercontent.com/23724310/167092015-95d1f559-1af1-4a29-b443-a01964836056.png">

Reply:
My fault - removed `Symfony\Bundle\FrameworkBundle\FrameworkBundle::class => ['all' => true]` from `bundles.php` after Symfony flex installation and everything is fine. Otherwise `aliases.yaml` ist not loaded.